src/Chapter1.hs

```diff
@@ -508,7 +510,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) < (abs y) then x else y
```

Comment:
`()` are redundant here. Usually, it's better to remove them for cleaner code. But I see that they can make code harder to read in the beginning...
```suggestion
closestToZero x y = if abs x < abs y then x else y
```